polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -34,6 +34,8 @@ export interface FrameProps {
   logo?: Logo;
   /** A horizontal offset that pushes the frame to the right, leaving empty space on the left */
   offset?: string;
+  /** Reduces the frame width */
+  shrink?: string;
```

Comment:
If this is still something you want to pursue I’d be more inclined to `offsetRight`. Going further I think updating `offset` to also accept a tuple or object with `start` and `end` values and updating the existing offset to use [margin-inline-start](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline-start) instead of margin-left.

If not, could you close the PR?